shellbase-core/src/main/scala/com/sumologic/shellbase/ShellCommandSet.scala

```diff
@@ -136,17 +138,26 @@ class ShellCommandSet(name: String, helpText: String, aliases: List[String] = Li
     })
   }
 
-  protected def printHelp(args: List[String]): Boolean = {
+  protected def printHelp(args: List[String], showAllCommands: Boolean): Boolean = {
```

Comment:
Sanity checked:  We haven't released a version with this method, so this is fine.

(One more reason for the 1.1 bump though)